### PR TITLE
feat: ITV-27 remove rooms page

### PR DESCRIPTION
**Pull request by LBHABEBIC**
remove rooms page and all references to it
https://hackney.atlassian.net/jira/software/projects/ITV/boards/40?selectedIssue=ITV-27

### CODE REVIEW

**Review by LBH-wgreeff — APPROVED** [review]
looks good to me

File: steps/household/rent.tsx [hunk 1]
```diff
@@ -150,6 +166,28 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "household"> = {
...
+            renderValue(notes: Notes): React.ReactNode {
+              if (notes.length === 0) {
+                return;
+              }
+
+              return <ReviewNotes notes={notes} />;
+            },
+          },
+        },
+      },
```
Comment (mrmartineau) [inline steps/household/rent.tsx:190]: @LBHABEBIC are these changes supposed to be here? are they from that blocked ticket that you have?

**Review by mrmartineau — CHANGES_REQUESTED** [review]
@LBHABEBIC it all looks good but I think you have a bunch of changes from another ticket/branch. If you do have code that isn't supposed to be hear, I suggest you close this PR and create a new one. If not, let me know and I will re-review this PR.

File: steps/household/rent.tsx (hunk 1 above)
Comment (LBHABEBIC) [reply steps/household/rent.tsx:190]: No they are not meant to be here. darn it.

**Pull request closed without merging**